src/components/Filter.js

```diff
@@ -0,0 +1,26 @@
+import { useCart } from "../context/cartContext";
+import "../styles/filter.css";
+export default function Filter() {
+  const { filteredData, dispatchData } = useCart();
+  return (
+    <nav className="filter-nav">
+      Sort By Price
+      <label>
+        <input
+          type="radio"
+          name="sort"
+          onChange={() => dispatchData({ type: "highToLow", filteredData })}

```

Comment:
its good to use capitalization here , `type:HIGH_TO_LOW`

Reply:
I have fixed it